docs/alternator/new-apis.md

```diff
@@ -181,27 +181,30 @@ entire data center, or other data centers, in that case.
 
 ## Tablets
 "Tablets" are ScyllaDB's new approach to replicating data across a cluster.
-It replaces the older approach which was named "vnodes". Compared to vnodes,
-tablets are smaller pieces of tables that are easier to move between nodes,
-and allow for faster growing or shrinking of the cluster when needed.
+It replaces the older approach which was named "vnodes". See
+[Data Distribution with Tablets](../architecture/tablets.rst) for details.
```

Comment:
Ok, but why did you decide to remove my two-line explanation of what tablets are?
I agree that a few years from now, when tablets are the default (or even the only way), the Alternator documentation doesn't need to explain tablets at all. But we're not there yet, and this is why this section even exists in this document.

Reply:
I have a pet peeve about  people claiming that because documentation can get out of sync, it shouldn't be written at all.
But I think it's good enough for now.